README.md

```diff
@@ -44,6 +45,29 @@ In order to be sure to use the correct name, there is a a public constant in
 `CassandraMigrationAutoConfiguration` that is called `CQL_SESSION_BEAN_NAME`. You can use that
 when declaring the session bean.
 
+### Reactive cassandra driver:
+There is no manually-defined CqlSession if you are using ReactiveCassandraConfiguration
+```
+@Configuration
+@RequiredArgsConstructor
+@EnableReactiveCassandraRepositories
+public class CassandraConfig extends AbstractReactiveCassandraConfiguration {
+....
+}
+```
+Following trick could be used to mark existing CqlSession as primary while you still need secondary session for Cassandra Migration.
```

Comment:
Maybe add something like "assuming your applications cassandra session bean is called 'cassandraSession'" to it is more obvious.